dpctl/tests/test_sycl_program.py

```diff
@@ -87,10 +87,10 @@ class TestProgramForLevel0GPU(unittest.TestCase):
 
     # Level zero program creation from a SPIR-V binary is not supported
     # on Windows.
-    @unittest.skipIf(
-        sys.platform in ["win32", "cygwin"],
-        "Level Zero module creation unsupported on Windows.",
-    )
+    # @unittest.skipIf(
+    #     sys.platform in ["win32", "cygwin"],
+    #     "Level Zero module creation unsupported on Windows.",
+    # )
```

Comment:
Please remove the unused code, not comment it. Git is already keeping this code forever.